src/main.tsx

```diff
@@ -10,19 +10,22 @@ import DayViewContextProvider from './data/contexts/DayViewContext';
 import { InstallPromptProvider } from './data/contexts/InstallPromptContext';
 import { generateData } from './data/MOCK_DATA';
 import { routesConfig } from './routes/routes';
+import ActivityContextProvider from './data/contexts/ActivitiesContext';
 
 if (import.meta.env.DEV) generateData();
 
 const router = createBrowserRouter([routesConfig]);
 
 ReactDOM.createRoot(document.getElementById('root') as HTMLElement).render(
+  <ActivityContextProvider>
```

Comment:
Definitely not what I meant for you to do when I said "make these into a context". Undo pls